tslint.json

```diff
@@ -1,16 +1,53 @@
 {
-  "extends": "tslint-no-circular-imports",
+  "//": "Consider using no-identical-functions and disabling in faulty tests",
```

Comment:
Why not directly use json comment in this json file instead of the bunch of tricky json keys? The current tslint is actually breaks some [RFCs](https://tools.ietf.org/html/rfc7159#section-4) and trigger unnecessary warning about duplicate keys problem in some IDEs.

Reply:
JSON does not support comments. TSLint would not be able to read this file if there was comments in it.

Reply:
@hansl it is true that json doesn’t support comments, but tslint does.

https://github.com/palantir/tslint/blob/master/test/configurationTests.ts

Reply:
Updated link above, as I copied the wrong one.